### ISSUE TITLE
Mouseover text fails to appear in images when placed in chat via markdown

### ISSUE THREAD

**Issue by ybur-yug** [gitterHQ/gitter#754]
It happens in any room I've tried with an the criteria below:

If you look up to the recent post that is an embedded image, I used the syntax:

`![alt text](link)`

with the alt text reading `RIP` but it fails to display if I mouseover in gitter, while the same embedded image in a README on github worked fine.

I am running Firefox 38.0 on Ubuntu 15.04.

I reached out on twitter and thought it was just a feature that was not implemented, but the response seemed to lean towards it being a bug [here](https://twitter.com/gitchat/status/617437443265052680).

**Comment by MadLittleMods** [comment]
Markdown syntax adds the `alt` attribute to the image which doesn't show a tooltip in the major browsers. It is alternate replacement if the image breaks and available to the accessibility tree.

The `title` attribute will do what you are expecting but it is standard across many markdown processors to only add only the `alt` attribute. [example gist](https://gist.github.com/anonymous/3603ec71340e6b886ccf)
